pom.xml

```diff
@@ -1406,6 +1382,59 @@
           <artifactId>maven-deploy-plugin</artifactId>
           <version>2.8.2</version>
         </plugin>
+
+	<!-- This plugin's configuration is used to store Eclipse m2e settings only. -->
+	<!-- It has no influence on the Maven build itself. -->
+	<plugin>
+	  <groupId>org.eclipse.m2e</groupId>
```

Comment:
What are the warnings that this can fix? or can these be integrated into the main build?